js/pro/binance.js

```diff
@@ -113,7 +114,22 @@ module.exports = class binance extends binanceRest {
     onClose (client, error) {
         this.options['streamBySubscriptionsHash'] = {};
         this.options['streamIndex'] = -1;
-        super.onClose (client, error);
+        if (client.error) {
```

Comment:
`super.onClose()` was giving a segmentation fault error in php, that I wasn't able to solve, so replaced for the code.